README.md

```diff
@@ -396,14 +397,59 @@ authClient.signIn({
 
 ### `signOut()`
 
-Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example).
+Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example) and clears all tokens stored in the `TokenManager`.
+
+By default:
+
+  - the user will be signed out using an API method 
+  - the page location will not be changed. 
+  
+When calling `signOut` with the default options, it is generally recommended to reload the page after the promise resolves to ensure your app is properly re-initialized in an unauthenticated state.
+
+Since the API method relies upon an XHR call and the ability to set 3rd-party cookies, it may fail in certain scenarios, leaving the user's remote session open. For this reason, we generally recommend using the redirect method over the API method. The redirect method sets the window location to Okta where the session and associated cookies will be fully cleared. See [Logout](https://developer.okta.com/docs/reference/api/oidc/#logout) in the OIDC API documentation for more details on the logout redirect method.
+
+`signOut` takes the following options:
+
+* `postLogoutRedirectUri` - (default: undefined) Enables the logout redirect method and sets the URI to which the browser will be redirected after logout. **The URI must be whitelisted** as a `Logout Redirect Uri` in your application's general settings. It is common to use your application's origin URI as the `postLogoutRedirectUri` so that users are sent to the application "home page" after signout.
+* `state` - (default: undefined) An optional value that is returned as a query parameter during the redirect to the `postLogoutRedirectUri`
+* `idToken` - (default: auto) Specifies the idToken. By default, `signOut` will look for a token named `idToken` within the `TokenManager`. If you have stored the idToken in a different location, you should retrieve it first and then pass it here. In any case where the `idToken` is not available, the user will be signed out using the API method.
+
 
 ```javascript
+// Sign out using the API
 authClient.signOut()
 .then(function() {
   console.log('successfully logged out');
 })
-.fail(function(err) {
+.catch(function(err) {
+  console.error(err);
+})
+.then(function() {
+  // Reload app in unauthenticated state
+  window.location.reload();
+});
+```
+
+
+```javascript
+// Sign out and redirect to Okta-hosted login page
+authClient.signOut({ redirect: true });
```

Comment:
This `redirect` is no longer an option in the latest version of this PR, yes?

Reply:
good catch